customer-overview.mdx

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Routing is broken and can't reach this page... it looks like it shares the same route with the other "overview" page, so nav shows both highlighted, but renders the other one
![image](https://github.com/monterey-ai/docs/assets/2603478/6abd401a-abe1-480c-aeda-9b2cbc133634)